<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>m3u8d</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>m3u8的url</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_M3u8Url"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>保存位置</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_SaveDir"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_SaveDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>保存的文件名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_FileName">
        <property name="placeholderText">
         <string>all</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_SkipTsCountFromHead"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>输出文件格式</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>HostType</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="comboBox_HostType">
        <item>
         <property name="text">
          <string>apiv1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>apiv2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="comboBox_OutputFormat">
        <item>
         <property name="text">
          <string>mp4格式(使用ffmpeg合并分段视频)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ts格式(直接拼接分段视频)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>跳过前面几个TS</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_Insecure">
      <property name="text">
       <string>允许不安全的https请求</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>下载进度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_RunDownload">
        <property name="text">
         <string>开始下载</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
